pyproject.toml

```
@@ -166,3 +166,9 @@ extend_skip_glob = ["asdf/extern/*", ".eggs/*", ".tox/*"]
 [tool.ruff]
 line-length = 120
 extend-exclude = ["asdf/extern/*", "docs/*"]
+
+[tool.codespell]
+skip="*.pdf,*.fits,*.asdf,.tox,asdf/extern,build,./tags,.git,docs/_build"
+ignore-words-list="""
+fo,
+"""
```

should this be a TOML list?
```suggestion
ignore-words-list= [
  'fo',
]
```

the `codespell` repository uses this format:
```toml
#[tool.codespell]
#builtin = ["clear","rare","informal","usage","code","names"]
#ignore-words-list = ["uint"]
#skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
```
https://github.com/codespell-project/codespell/blob/f010d35ad322ea6fa189b20f747f87f1c3c4ecb2/pyproject.toml#L100

and the code seems to be able to parse a list:
```python
def parse_ignore_words_option(ignore_words_option: List[str]) -> Set[str]:
    ignore_words = set()
    if ignore_words_option:
        for comma_separated_words in ignore_words_option:
            for word in comma_separated_words.split(","):
                ignore_words.add(word.strip())
    return ignore_words
```
https://github.com/codespell-project/codespell/blob/f010d35ad322ea6fa189b20f747f87f1c3c4ecb2/codespell_lib/_codespell.py#L606-L612

---

This seems to not work, see https://results.pre-commit.ci/run/github/18112754/1670856541._AN1akuuRCOmMcZjPLbg4w

---

I apologize then, I guess that's why it's commented out in their repository

---

> should this be a TOML list?
> 
> the `codespell` repository uses this format:
> 
> ```toml
> #[tool.codespell]
> #builtin = ["clear","rare","informal","usage","code","names"]
> #ignore-words-list = ["uint"]
> #skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
> ```
> 
> [codespell-project/codespell@`f010d35`/pyproject.toml#L100](https://github.com/codespell-project/codespell/blob/f010d35ad322ea6fa189b20f747f87f1c3c4ecb2/pyproject.toml#L100)

You are mistaken, all of that is commented out in the toml file. What it is actually using is here: https://github.com/codespell-project/codespell/blob/075a51742333a1ed9d35a3f50ab27f1ebe082448/pyproject-codespell.precommit-toml#L1-L7

```toml
[tool.codespell]
#builtin = ["clear","rare","informal","usage","code","names"]
builtin = "clear,rare,informal,usage,code,names"
#ignore-words-list = ["uint"]
ignore-words-list = "adn,master,uint"
#skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
skip="./.*,codespell_lib/data/*,codespell_lib/tests/*"
```
where it is doing everything using a single string.